staging/src/k8s.io/apiserver/pkg/server/healthz/healthz_test.go

```diff
@@ -44,26 +44,13 @@ func TestInstallHandler(t *testing.T) {
 
 func TestInstallPathHandler(t *testing.T) {
 	mux := http.NewServeMux()
-	InstallPathHandler(mux, "/healthz/test")
```

Comment:
Seems like this should remain to ensure that the handler doesn't register anything to to healthz.  If it did this test woudl panic

Reply:
You want this to be added so that the function tests registering to two separate paths?

Reply:
@deads2k